misk-launchdarkly-core/src/test/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlagsTest.kt

```diff
@@ -100,6 +103,28 @@ internal class LaunchDarklyFeatureFlagsTest {
     }
   }
 
+  data class JsonFeature(val value: String)
+
+  @Test
+  fun getJson() {
+    val json = """{
+      "value" : "dino"
+    }""".trimIndent()
+    // LD uses Gson internally, so we need to do the same for mocking.
+    val gson = Gson()
+    val jsonElement = gson.toJsonTree(gson.fromJson(json, JsonFeature::class.java))
+    @Suppress("DEPRECATION")
+    Mockito
+        .`when`(client.jsonValueVariationDetail(anyString(), any(LDUser::class.java),
```

Comment:
Nice, haven't run into much mockito usage yet in my misk project work

Reply:
not a fan on mockito, but I'm being consistent

Reply:
your other option would have been to implement 20+ methods just to mock out a single method on a fake :)

Reply:
pretty easy to do with Intellij, just defaults to `TODO` in kt or `UnsupportedOperationException` in Java

Reply:
But the clutter 😢